GooglePayment/src/main/java/com/braintreepayments/api/GooglePayment.java

```diff
@@ -119,21 +120,36 @@ public void onConfigurationFetched(Configuration configuration) {
                                 .setEnvironment(getEnvironment(configuration.getGooglePayment()))
                                 .build());
 
-                JSONObject json = new JSONObject();
-                JSONArray allowedCardNetworks = buildCardNetworks(fragment);
 
+                JSONObject json = new JSONObject();
                 try {
+                    JSONArray allowedCardNetworks = buildCardNetworks(fragment);
+                    JSONArray allowedPaymentMethods = new JSONArray()
+                            .put(new JSONObject()
+                                    .put("type", "CARD")
+                                    .put("parameters", new JSONObject()
+                                            .put("allowedAuthMethods", new JSONArray()
+                                                    .put("PAN_ONLY")
+                                                    .put("CRYPTOGRAM_3DS"))
+                                            .put("allowedCardNetworks", allowedCardNetworks)));
+
+                    String paypalClientId = configuration.getGooglePayment().getPaypalClientId();
+                    if (paypalClientId != null && !TextUtils.isEmpty(paypalClientId)) {
```

Comment:
```suggestion
                    if (!TextUtils.isEmpty(paypalClientId)) {
```

`TextUtils.isEmpty` does a null check, so you don't need `paypalClientId != null`.

Reply:
@sestevens ty 🙏 I was thinking it might but honestly didn't take the time to look. Good to keep in mind going forward